pkg/parser/validate/validate.go

```diff
@@ -1,13 +1,19 @@
 package validate
 
 import (
+	"github.com/CircleCI-Public/circleci-yaml-language-server/pkg/dockerhub"
 	"github.com/CircleCI-Public/circleci-yaml-language-server/pkg/parser"
 	"github.com/CircleCI-Public/circleci-yaml-language-server/pkg/utils"
 
 	"go.lsp.dev/protocol"
 )
 
+type ValidateAPIs struct {
+	DockerHub dockerhub.DockerHubAPI
+}
+
 type Validate struct {
+	APIs        ValidateAPIs
```

Comment:
why plural ? we can have multiple apis ?

Reply:
Yes, atm there is only the DockerHub API we can imagine having the same logic for the CCI API